### ISSUE TITLE
How can I make the sensor reading board?

### ISSUE THREAD

**Issue by Der-mark** [binghao-huang/3D-ViTac_Tactile_Hardware#2]

**Comment by Der-mark** [comment]
![image](https://github.com/user-attachments/assets/9edaffb1-7d7c-4ecb-88f5-6b86c448c186)
![image](https://github.com/user-attachments/assets/62ece4a7-9e36-4eb8-9712-6afdf653957d)
When I uploaded the zip file downloaded from the Gerber-connector in the picture, I only parsed the connnector and not the sensorboard.

**Comment by Der-mark** [comment]
oh ! I found the reading board gerber file ! thanks!

**Comment by binghao-huang** [comment]
No worry! Feel free to ask questions if you have any issues. By the way, it will be great if you share a video of your final sensor when you complete it. I am pretty curious about the reproduction of the sensor in other lab.

**Comment by Der-mark** [comment]
https://github.com/user-attachments/assets/d133f686-5053-4a85-bca0-50b34afc397d


https://github.com/user-attachments/assets/135e22a2-82c6-473b-8e82-5fcc3eff3934



Thank you very much. Your teaching video is very detailed and professional. I couldn't find the gerber file at first, so I learned to draw a PCB by myself. Although I didn't use it later, I also learned SMT patch technology through this project. In addition, the soft stainless steel yarn is also very novel to me. The final effect of this tactile sensor is amazing. Thank you for your work and help! I want to have fun with it.
